### ISSUE TITLE
`no-standalone-expect`: False positive when there are multiple `describe` blocks

### ISSUE THREAD

**Issue by michaelfaith** [playwright-community/eslint-plugin-playwright#291]
If using nested `describe` blocks to organize multiple sets of tests, the `expect` calls within the `it` tests in the `describe` blocks after the first, are causing `no-standalone-expect` to flag them incorrectly.  

```typescript
describe('base', () => {
  it('top level test', async () => {
    expect... // << this one doesn't flag
  });

  describe('nested', () => {
    it('nested test', async () => {
      expect... // << this one flags no-standalone-expect (false positive)
    });
  });
});
```

Note: if the test function is `test` instead of `it`, then it also doesn't flag.  It only seems to impact tests that use `it`

**Comment by mskelton** [comment]
Interesting, sounds like an issue with the aliases not applying in all instances for some reason.

**Comment by mskelton** [comment]
@michaelfaith I was unable to reproduce this. Could you create a reproduction code sandbox?

**Comment by mskelton** [comment]
Closing due to lack of reproduction steps

**Comment by michaelfaith** [comment]
@mskelton  Sorry for not responding sooner.  I can upload a reproduction, but a really simple recreation that might help shows how simply aliasing the `test` import breaks this test:

![image](https://github.com/playwright-community/eslint-plugin-playwright/assets/8071845/3306887a-5d8b-4e23-a139-d4674544ff8a)

**Comment by michaelfaith** [comment]
It's also worth noting that it broke the `expect-expect` test too.  That "basic test without expect" `it` should have a violation that it didn't report here.

**Comment by mskelton** [comment]
@michaelfaith I'm guessing you didn't configure global aliases in your config.

https://github.com/playwright-community/eslint-plugin-playwright#aliased-playwright-globals

**Comment by michaelfaith** [comment]
> @michaelfaith I'm guessing you didn't configure global aliases in your config.
> 
> https://github.com/playwright-community/eslint-plugin-playwright#aliased-playwright-globals

I did not.  Is that necessary even for local import aliases, like this?  I confirmed that if I add the following, it works as expected.

```js
settings: {
  playwright: {
    globalAliases: {
      test: ['it'],
    },
  },
},
```

**Comment by mskelton** [comment]
@michaelfaith Yes, it's necessary even with import aliases. The primary reason for this is that imports often can come from files which define fixtures, so the import source isn't always just `@playwright/test`, so auto-detecting it is not something that can always be done.
